src/templates/partials/theader.html

```diff
@@ -45,24 +38,33 @@
         <div id="navMenu" class="navbar-menu">
             <div class="navbar-start">
                 {% if user.is_authenticated %}
-                    <div class="navbar-item has-dropdown">
-                        <a class="navbar-link">View</a>
-                        <div class="navbar-dropdown">
-                            <a href="{% url 'recipient_dashboard' %}" class="navbar-item">Available Donations</a>
-                            <a href="{% url 'donor_dashboard:org_list' %}" class="navbar-item">My Organizations</a>
-                        </div>
-                    </div>
+                    <a class="navbar-item" href="{% url 'recipient_dashboard' %}">
+                        <span class="icon is-small"><i class="fa fa-home" aria-hidden="true"></i></span>
+                        <span>Dashboard</span>
+                    </a>
+                    <a class="navbar-item" href="{% url 'recipient_orders' %}">
+                        <span class="icon is-small"><i class="fa fa-list" aria-hidden="true"></i></span>
+                        <span>Reservations</span>
+                    </a>
+                    <a class="navbar-item" href="{% url 'donor_dashboard:org_list' %}">
+                        <span class="icon is-small"><i class="fa fa-group" aria-hidden="true"></i></span>
+                        Organizations
+                    </a>
                 {% endif %}
             </div>
             <div class="navbar-end">
                 {% if user.is_authenticated %}
-                    <a class="navbar-item" href="{% url 'recipient_dashboard' %}">Dashboard</a>
-                    <a class="navbar-item" href="{% url 'recipient_orders' %}">Reservations</a>
-                    <a class="navbar-item" href="{% url 'user_profile:profile' %}">Profile</a>
+                    <a class="navbar-item" href="{% url 'user_profile:profile' %}">
+                        <span class="icon is-small"><i class="fa fa-user" aria-hidden="true"></i></span>
+                        <span>Profile</span>
+                    </a>
                 {% endif %}
                 <div class="navbar-item">
                     {% if user.is_authenticated %}
-                        <a class="button is-outlined is-dark" href="{% url 'messaging:messaging' %}"><span>Messages</span></a>
+                        <a class="button is-outlined is-dark" href="#">
```

Comment:
The messaging URL (href) is wrong here. I think it got over writeen. Can you please update to the latest?

Reply:
Fixed! Thanks for catching